packages/react-component-library/src/components/Modal/partials/StyledSecondaryButton.tsx

```diff
@@ -1,12 +1,5 @@
-import { selectors } from '@defencedigital/design-tokens'
 import styled from 'styled-components'
 
 import { StyledButton } from './StyledButton'
 
-const { mq } = selectors
-
-export const StyledSecondaryButton = styled(StyledButton)`
-  ${mq({ gte: 'xs' })`
-    margin-left: auto;
-  `}
-`
+export const StyledSecondaryButton = styled(StyledButton)``
```

Comment:
Is this partial redundant now? Could it be removed?

Reply:
Good question! This one is still being used in `StyledFooter.tsx`. But, while checking that, I found that Dialog was also trying to set mobile button padding on `StyledTertiaryButton`. That isn't needed now, and it was the last bit of styling on `StyledTertiaryButton` so that one has gone now :slightly_smiling_face:

Reply:
Good question! 

It seems so – I've swapped out the selector and removed `StyledSectoryButton` and all seems well.

Also, while checking that, I found that Dialog was also trying to set mobile button padding on `StyledTertiaryButton`. That isn't needed now, and it was the last bit of styling on `StyledTertiaryButton` so that one has gone now :slightly_smiling_face: